tox.ini

```diff
@@ -3,12 +3,21 @@ envlist =
     py{37,38,39,310,311}
     flake8
 
+[gh-actions]
+python =
+    3.7: py37
+    3.8: py38
+    3.9: py39
+    3.10: py310
+    3.11: py311, flake8
+
 [testenv]
 passenv=HOME
 sitepackages = False
 deps = -r{toxinidir}/requirements.txt
        -r{toxinidir}/test-requirements.txt
        -r{toxinidir}/dist-requirements.txt
+       py37: importlib-metadata<5.0
```

Comment:
question: If I understood correctly, this should not be needed once my supported versions patch merges, right?

Reply:
Correct, as soon as https://github.com/release-depot/flask-container-scaffold/pull/37 lands all of these changes will go away.